assets/airports/egcc.json

```diff
@@ -0,0 +1,582 @@
+ {
+  "name": "Manchester Airport",
+  "level": "medium",
+  "radio": {
+    "twr": "Manchester Delivery",
```

Comment:
Are you certain this is called "Manchester Delivery"? The one who tells them cleared to land and cleared for takeoff?